recipes/genie/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+import os, glob
+from conans import ConanFile, tools
+
+
+class GenieConan(ConanFile):
+    name = "genie"
+    license = "BSD-3-clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bkaradzic/GENie"
+    description = "Project generator tool"
+    topics = ("conan", "genie", "project", "generator", "build")
+    settings = "os", "arch", "compiler"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def build_requirements(self):
+        if self.settings.compiler == "Visual Studio":
+            self.build_requires("cccl/1.1")
+
+        if self.settings.os == "Windows" and tools.os_info.is_windows:
+            if "make" not in os.environ.get("CONAN_MAKE_PROGRAM", ""): # or not tools.which("make"):
+                self.build_requires("make/4.2.1")
```

Comment:
`make` packages a `gmake` executable which you don't call in `build`.
I think you should use `autotools.make` instead of `self.run("make")`, as the first uses `CONAN_MAKE_PROGRAM`.

Reply:
Perhaps this is an improve for a follow up PR?